commit f087d20e6e60a49c756936b4312f5d194d8e63b4
parent a8e9513783f335b1ac7255e40a663adfffc4b475
Author: Chris Down <chris@chrisdown.name>
Date:   Wed, 22 Apr 2020 15:48:08 +0100

getatomprop: Add forward declaration

No functional changes, but for every other function we have a forward
declaration here. getatomprop should be no exception.

Diffstat:
M	dwm.c	|	1	+
1 file changed, 1 insertion(+), 0 deletions(-)
diff --git a/dwm.c b/dwm.c
@@ -169,6 +169,7 @@ static void focus(Client *c);
 static void focusin(XEvent *e);
 static void focusmon(const Arg *arg);
 static void focusstack(const Arg *arg);
+static Atom getatomprop(Client *c, Atom prop);
 static int getrootptr(int *x, int *y);
 static long getstate(Window w);
 static int gettextprop(Window w, Atom atom, char *text, unsigned int size);
